app/src/main/java/fr/groggy/racecontrol/tv/ui/home/HomeActivity.kt

```diff
@@ -50,6 +60,93 @@ class HomeActivity : FragmentActivity(R.layout.activity_home) {
         findViewById<View>(R.id.settings).setOnClickListener {
             startActivity(SettingsActivity.intent(this))
         }
+
+        val currentVersion = BuildConfig.VERSION_NAME // Getting current version (eg. comparing with Version)
+        // val currentVersion = "2.6.2" // Debug
+        val currentVersionText: TextView = findViewById<TextView>(R.id.currentVersionText)
+        currentVersionText.text = currentVersion // Showing Version number top left
+
+        AsyncTask.execute {
+
+            val httpsURL =
+                "https://raw.githubusercontent.com/leonardoxh/race-control-tv/master/app/build.gradle.kts"
+            val myUrl = URL(httpsURL)
```

Comment:
Can you move this entire thing to a separate class like `updateService` ?

Reply:
And also use coroutines to handle this network part.

Reply:
Sure, I can do that. 
I thought about planning it like this: Merging and publishing a new release with this current version of the updater so the users have a general update service, so they don't need to manually sideload the new version every time.

After this I'll open a new PR to refine the code and put them into another class and also convert the ThreadWorker to Coroutines. This then can be updated into another release when there are other significant changes to the app.

Do we wanna do it like this?

Reply:
I think it is better to put all of this in 1 PR.

Because this async task work and manual work seems like 2010 :D Also `AsyncTask` is deprecated...

Reply:
And second... As the app is pretty much dead (no live content yet) I'd say no rush on this one.

Reply:
Okay so I have played around with Coroutines a bit and I need to do some research on this, since I only worked on small projects and with Async tasks so far. So this might take a while.